decidim-meetings/db/migrate/20210602080854_rename_events_to_meetings.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class RenameEventsToMeetings < ActiveRecord::Migration[6.0]
+  def change
+    Decidim::ContentBlock.where(manifest_name: :upcoming_events).find_each do |block|
```

Comment:
How about using `.update_all` so we can skip models validations instead of `.find_each` and `.save`?